### PR TITLE
Call bindShaders from addSceneObject

### PR DESCRIPTION
**Pull request by NolaDonato**
When a scene object is added at the top level, GVRScene.bindShaders is automatically called if necessary. Now the remote scripting demo doesn't have to explicitly call bindShaders.
clear light list when clearing entire scene
return if texture is null in custom shader (instead of crashing)

### CODE REVIEW

**Comment by NolaDonato** [comment]
Adding Missing DCO GearVRf-DCO-1.0-Signed-off-by: Nola Donato nola.donat@samsung.com

File: GVRf/Framework/framework/src/main/java/org/gearvrf/IAssetEvents.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package org.gearvrf;
```
Comment (thomasflynn) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/IAssetEvents.java]: needs copyright header

Comment (NolaDonato) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/IAssetEvents.java]: removed this file, not needed

**Comment by thomasflynn** [comment]
fails to build.  Apparently GVRContext and GVRImporter reference IAssets in a number of places.

**Comment by NolaDonato** [comment]
I reverted GVRContext, GVRImportSettings, GVRImporter to master, it builds now and I tested with gvr-remote-scripting demo which no longer needs bindShaders

**Pull request merged**